custom/lib_portal/models/settings.py

```diff
@@ -0,0 +1,36 @@
+from odoo import models, fields, api,  _
+
+
+class LibrarySettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    email = fields.Char(string='Common Mail Address',
+                        default='magdeburgcitylibrary@example.com')
+    lend_duration = fields.Char(string='Lend Period', default=30)
+    lend_limit = fields.Char(string='Lend Limit', default=4)
+
+    def set_values(self):
+        res = super(LibrarySettings, self).set_values()
+        self.env['ir.config_parameter'].set_param(
+            'lib_portal.email', self.email)
+        self.env['ir.config_parameter'].set_param(
+            'lib_portal.lend_duration', self.lend_duration)
+        self.env['ir.config_parameter'].set_param(
+            'lib_portal.lend_limit', self.lend_limit)
```

Comment:
it's better to define Env before using it if you will use it more than one time

```suggestion
        param_obj = self.env['ir.config_parameter']
        param_obj.set_param(
            'lib_portal.email', self.email)
        param_obj.set_param(
            'lib_portal.lend_duration', self.lend_duration)
        param_obj.set_param(
            'lib_portal.lend_limit', self.lend_limit)
```